packages/core-assortments/jest.config.js

```diff
@@ -1,5 +1,14 @@
 /** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
 export default {
-  preset: 'ts-jest',
+  preset: '@shelf/jest-mongodb',
```

Comment:
Unit tests should not test the db, that is beeing done with the integration tests, only test pure functions

Reply:
A third example of code is the file build-paths.ts. It contains only pure functions but takes functions as an argument. Such a code is unit testable as well because we can supply a stub function that usually involves db calls:

```
const paths = await walkUpFromAssortment({
  resolveAssortmentLink: (assortmentId) => {
    if (assortmentId === 'subsubchild') {
      return {
        assortmentId,
        parentIds: ['subchild'],
      };
    }
    if (assortmentId === 'subchild') {
      return {
        childAssortmentId: assortmentId,
        parentIds: ['root'],
      };
    }
    if (assortmentId === 'root') {
      return {
        childAssortmentId: assortmentId,
        parentIds: [],
      };
    }
    return null;
  },
  assortmentId: 'subsubchild',
});

expect(paths).toEqual([{ links: ['subsubchild', 'subchild', 'root'] }]);
```

Reply:
@Mikearaya